lib/OA/Dal/Delivery.php

```diff
@@ -26,6 +26,9 @@
 ###END_STRIP_DELIVERY
 $GLOBALS['_MAX']['FILES'][$file] = true;
 
+function OA_Dal_Delivery_isValidResult($result) {
```

Comment:
Nice job, encapsulating stuff. As long as PHP remains lacy (type-hinting) this will work ...